openpype/hosts/maya/api/lib_rendersettings.py

```diff
@@ -96,11 +97,7 @@ def set_default_renderer_settings(self, renderer):
 
     def _set_Arnold_settings(self, settings, width, height):
         """Sets settings for Arnold."""
-        mel.eval('unifiedRenderGlobalsWindow;')
-
-        if cmds.window("unifiedRenderGlobalsWindow", exists=True):
-            cmds.deleteUI("unifiedRenderGlobalsWindow")
-
+        createOptions()
```

Comment:
Can we move the `mtoa` import here? Technically you can install Maya without Arnold so I'd rather not have it required by forcing the import on top level.

Also, since this file also has other renderers implemented I'd avoid importing `createOptions` into this module but instead just do `import mtoa.core` and `mtoa.core.createOptions()` to remain explicit.

Additionally, be aware that the call to `createOptions` required the mtoa plug-in to currently be loaded in the Plug-in Manager.

Reply:
Thank you Roy those are good points I'll do that!